firebase-firestore/src/main/java/com/google/firebase/firestore/util/CustomClassMapper.java

```diff
@@ -66,18 +66,35 @@ private static void hardAssert(boolean assertion, String message) {
     }
   }
 
+  /**
+   * Returns a mapper with a given document ID. The document ID will be used to populate the {@link
```

Comment:
nit: "with" => "that's associated with"  (just seems clearer to me)

Reply:
No longer needed.